tests/errors/cfuncptr.pyx

```diff
@@ -26,11 +30,25 @@ def fail_exceptstar(bad):
     fptr_b = exceptstar
     fptr_c = exceptstar
 
+cdef int cb(int param) nogil:
+    return param
+
+cdef void cb_void(int param) except * nogil:
+    return
+
+def fail_struct_pointer():
+    cdef mystruct ms = mystruct(&cb, &cb_void)
+
+
 _ERRORS = """
 13:13: Cannot assign type 'int (int) except? -2' to 'int (*)(int) except -2'
 14:13: Cannot assign type 'int (int) except? -2' to 'int (*)(int) except -1'
 15:13: Cannot assign type 'int (int) except? -2' to 'int (*)(int) except? -1'
-25:13: Cannot assign type 'int (int) except *' to 'int (*)(int) noexcept'
-26:13: Cannot assign type 'int (int) except *' to 'int (*)(int) except -1'
-27:13: Cannot assign type 'int (int) except *' to 'int (*)(int) except? -1'
+29:13: Cannot assign type 'int (int) except *' to 'int (*)(int) noexcept'
+30:13: Cannot assign type 'int (int) except *' to 'int (*)(int) except -1'
+31:13: Cannot assign type 'int (int) except *' to 'int (*)(int) except? -1'
+40:32: Cannot assign type 'int (*)(int) except? -1 nogil' to 'int (*)(int) noexcept nogil'
+40:32: Cannot assign type 'int (*)(int) except? -1 nogil' to 'int (*)(int) noexcept nogil'
```

Comment:
I don't know why those lines needs to be duplicated. When redundant lines are removed, the test is failing...